schemas/oic.wk.p-schema.json

```diff
@@ -95,6 +95,16 @@
           "maxLength": 64,
           "readOnly": true,
           "description": "Manufacturer's defined information for the platform. The content is freeform, with population rules up to the manufacturer"
+        },
+       "nct": {
```

Comment:
The Property Name in the CR is "mnnct" not "nct"  Which one is right?

Reply:
sorry, the property name should be changed to "mnnct" to align with the CR